content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -321,18 +322,24 @@ $ helm template install/kubernetes/helm/istio <the flags you used to install Ist
 
 ## Install Istio with access to all external services by default
 
-An alternative to calling external services directly is to instruct the Istio proxy to pass through the calls to all the
-external services. This option allows you to start evaluating Istio quickly, without controlling access to external
+An alternative to calling external services directly is to instruct the Istio proxy to pass through, instead
+of block, calls to external services by default. This option allows you to start evaluating Istio quickly,
+without controlling access to external
 services, and decide to [configure access to external services](#controlled-access-to-external-services) later.
 
-To allow access to any external service on any ports without an HTTP service or a service entry within the mesh,
-Istio has an [installation option](/docs/reference/config/installation-options/).
-For example, if you don't register an HTTP service or define a service entry for the 8000
-port within the mesh, you can configure the sidecar proxy to pass the request to any external service on that port.
+Istio has an [installation option](/docs/reference/config/installation-options/)
```

Comment:
I think it was a comment of @rcaballeromx to swap the order in the sentence.

Reply:
It was, because the fact that the option exists is not as important as its purpose. What do you want to emphasize, the link to all the installation options or why should the reader look for the option?